pkgs/games/steam/linux-steam-integration.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig, git, pkgs, gettext,
+  gcc_multi, libressl }:
+
+let
+  version = "0.7.2";
+  steamBinPath = "${stdenv.lib.makeBinPath (with pkgs; [ steam ])}/steam";
+  zenityBinPath = "${stdenv.lib.makeBinPath (with pkgs; [ gnome3.zenity ])}/zenity";
+
+in stdenv.mkDerivation rec {
+  name = "linux-steam-integration-${version}";
+
+  nativeBuildInputs = [ meson ninja pkgconfig git gettext gcc_multi ];
+  buildInputs = [ pkgs.gtk3 libressl ];
```

Comment:
It is better to take `gtk3` as an argument rather than using `gtk3` from the `pkgs` attribute set.